.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2.1
+
+jobs:
+  build-and-test:
```

Comment:
```suggestion
  install-and-test:
```

It does build, but that's because that's part of running the tests.

Reply:
Makes sense - changed.